traffic_ops/traffic_ops_golang/ats/atsserver/ipallowdotconfig.go

```diff
@@ -1,148 +0,0 @@
-package atsserver
```

Comment:
Yeah, that was the plan, but you asked me to delete the routes?

Reply:
Yeah, I wasn't really clear about wanting to do that in a separate PR. I'm just trying to throw out ideas for breaking up PRs in the future, since this PR is doing way too many things at this point.